frameworks/CSharp/aspnet/benchmark_config.json

```diff
@@ -203,7 +203,7 @@
       "notes": "",
       "versus": ""
     },
-    "mono": {
+    "default": {
```

Comment:
If there is a bug (e.g. getting a stack trace instead of a nicely printed error when trying to run a windows test on linux or vise versa), then that should be an issue that we resolve in the toolset. The naming convention from before your changes is, however, correct.